### PR TITLE
Created a new page that contains all logged in user archives.

### PR DESCRIPTION
**Pull request by affan00**

### CODE REVIEW

File: app/controllers/archives_controller.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class ArchivesController < ApplicationController
+    before_action :authenticate_user!
+    include SessionsHelper
+    include ApplicationHelper
+  
+    def index
+      @cameras = load_user_cameras(true, false)
+      api = get_evercam_api
+      @camera = api.get_camera(params[:id], true) if @camera.nil?
```
Comment (azharmalik3) [inline app/controllers/archives_controller.rb]: Why? Do you really need this?

Comment (affan00) [reply app/controllers/archives_controller.rb]: I need this for getting the @offset value.

Comment (azharmalik3) [reply app/controllers/archives_controller.rb]: 1) But this page will contain the list of all archives then how you get camera-id `params[:id]`?

2) Does it mean you will use one camera offset for all archives?

Comment (affan00) [reply app/controllers/archives_controller.rb]: Yes, I got your point, that is a mistake, I will fix this.

File: app/controllers/archives_controller.rb [hunk 2]
```diff
@@ -6,11 +6,5 @@ class ArchivesController < ApplicationController
     def index
       @cameras = load_user_cameras(true, false)
       api = get_evercam_api
```
Comment (azharmalik3) [inline app/controllers/archives_controller.rb]: Don't need this.

File: app/controllers/archives_controller.rb (hunk 1 above)
Comment (affan00) [reply app/controllers/archives_controller.rb]: @azharmalik3 used the archive created at date for getting the offset value and removed the offset code from the archive controller.

**Pull request merged**